src/Java/Lesson5/CivilAviation.java

```diff
@@ -0,0 +1,43 @@
+package Java.Lesson5;
+
+import java.util.Scanner;
+
+public class CivilAviation extends AirTransport {
+    private static int passengers;
```

Comment:
@Ihar-Malinouski почему статичный? + переименуй, чтобы было понятней что за переменная: 
```suggestion
    private int incomingPassengers;
```